Lorenz scraper not working

**Describe the bug**
Lorenz scraper produces an error: victim_name = victim.find("div", class_="panel-heading").find("h3").text.strip()
\ AttributeError: 'NoneType' object has no attribute 'text'

**To Reproduce**
1. Input right Lorenz site URL
2. Run ransomwatch task

**Expected behavior**
Scraper works on Lorenz.

**Screenshots**
![image](https://user-images.githubusercontent.com/45006777/143705825-c3b98816-1e91-4038-a1f3-342ffec48de7.png)

**Logs**
app_1    | 2021/11/27 18:10:08 [ERROR] Got an error while scraping Lorenz, notifying
app_1    | 2021/11/27 18:10:08 [ERROR] Traceback (most recent call last):
app_1    |   File "/app/ransomwatch.py", line 77, in main
app_1    |     s.scrape_victims()
app_1    |   File "/app/sites/lorenz.py", line 67, in scrape_victims
app_1    |     self._handle_page(r.content.decode())
app_1    |   File "/app/sites/lorenz.py", line 19, in _handle_page
app_1    |     victim_name = victim.find("div", class_="panel-heading").find("h3").text.strip()
app_1    | AttributeError: 'NoneType' object has no attribute 'text'

**Environment**
 - OS: Linux Debian 10
 - How you are running it: Docker local image

**Additional context**
-

--- a/src/sites/lorenz.py
+++ b/src/sites/lorenz.py
@@ -13,11 +13,15 @@ class Lorenz(SiteCrawler):
 
     def _handle_page(self, body: str):
         soup = BeautifulSoup(body, "html.parser")
-        victim_list = soup.find_all("div", {"id" : re.compile('comp.*')})
+        victim_list = soup.find_all("div", {"id" : re.compile("comp.*")})
 
         for victim in victim_list:
-            victim_name = victim.find("div", class_="panel-heading").find("h3").text.strip()
-            victim_leak_site = self.url + "/#" + victim.get('id')
+            victim_h3 = victim.find("div", class_="panel-heading").find("h3")
+            if victim_h3 is None:
+                # unpublished victims are in a h4
+                continue
+            victim_name = victim_h3.text.strip()
+            victim_leak_site = self.url + "/#" + victim.get("id")
 
             if victim.find("span", class_="glyphicon"):
                 published = victim.find("span", class_="glyphicon").next_sibling
@@ -52,7 +56,6 @@ class Lorenz(SiteCrawler):
 
             # find all pages
             page_nav = soup.find_all("a", class_="page-numbers")
-
             
             site_list = []
             site_list.append(self.url)

